### ISSUE TITLE
Ubuntu22 libssl version changed

### ISSUE THREAD

**Issue by cengelenVW** [actions/runner-images#9429]
### Description

https://github.com/actions/runner-images/blob/main/images/ubuntu/scripts/build/install-sqlpackage.sh
the ubuntu22 libssl package version is changed now:
http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.22_amd64.deb

The version changed again after fixing issue #9309, so that's why I created a new issue.

### Platforms affected

- [ ] Azure DevOps
- [ ] GitHub Actions - Standard Runners
- [ ] GitHub Actions - Larger Runners

### Runner images affected

- [ ] Ubuntu 20.04
- [X] Ubuntu 22.04
- [ ] macOS 11
- [ ] macOS 12
- [ ] macOS 13
- [ ] macOS 13 Arm64
- [ ] macOS 14
- [ ] macOS 14 Arm64
- [ ] Windows Server 2019
- [ ] Windows Server 2022

### Image version and build link

Any ubuntu22 stable release

### Is it regression?

no

### Expected behavior

Download the current version

### Actual behavior

Tries to download a non-existant version

### Repro steps

Build any ubuntu22 recent release

**Comment by shamil-mubarakshin** [comment]
Hey @cengelenVW, thanks for reporting.
https://github.com/actions/runner-images/pull/9428 is being tested. We will let you know once fix is merged

**Comment by shamil-mubarakshin** [comment]
@cengelenVW, PR with fix has been merged. Please sync main branch
